examples/sql_bank/main.go

```diff
@@ -49,10 +52,15 @@ func moveMoney(db *sql.DB) {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if _, err = tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
-			log.Fatal(err)
-		}
+		startTime := time.Now()
+		// TODO(vivek): Query is very slow, needs investigation.
```

Comment:
We don't optimize `IN` queries right now. See #2140. You could rewrite this as:

```
SELECT id, balance from accounts WHERE id = $1 OR id = $2
```

I'm planning on getting the "point" query support done this week.